nix/default.nix

```diff
@@ -1,4 +1,4 @@
-{ system ? builtins.currentSystem }:
+{ system ? builtins.currentSystem, disableSystemd ? true }:
```

Comment:
let's switch the default to disableSystemd = false

Reply:
done